pkg/reconciler/mongodb/resources/receive_adapter.go

```diff
@@ -136,7 +136,7 @@ func makeEnv(args *ReceiveAdapterArgs) ([]corev1.EnvVar, error) {
 		Value: "/etc/mongodb-credentials",
 	}}
 
-	envs = append(envs, args.Configs.ToEnvVars()...)
```

Comment:
This is the question I had

Reply:
I get a nil pointer exception probably because it in the test file I do not have a field Configs but Im not sure how to create it?

Reply:
OK, yes, this shouldn't be removed...

You should do something like ApiServerSource is doing. 
https://github.com/knative/eventing/blob/5632b0975a73d4c9196da928ceb9216654284c67/pkg/reconciler/apiserversource/resources/receive_adapter_test.go#L143

And this is how you create them: https://github.com/knative/eventing/blob/5632b0975a73d4c9196da928ceb9216654284c67/pkg/reconciler/apiserversource/resources/receive_adapter_test.go#L192

Add the ceOverrides as well if you can: https://github.com/knative/eventing/blob/5632b0975a73d4c9196da928ceb9216654284c67/pkg/reconciler/apiserversource/resources/receive_adapter_test.go#L160